R/plot.parameters_model.R

```diff
@@ -197,35 +309,95 @@ plot.see_parameters_model <- function(x,
     # plot setup for metafor-objects
     p <- ggplot(x, aes(y = .data$Parameter, x = .data$Coefficient, color = .data$group)) +
       geom_vline(aes(xintercept = y_intercept), linetype = "dotted") +
-      geom_pointrange(aes(xmin = .data$CI_low, xmax = .data$CI_high), size = size_point, fatten = x$size_point, shape = x$shape) +
       theme_modern(legend.position = "none") +
       scale_color_material() +
       guides(color = "none", size = "none", shape = "none")
 
-  } else if (sum(grepl("^CI_low", colnames(x))) > 1) {
+    if (show_density) {
+      # p <- p + density_layer
+      message(
+        insight::format_message("Plotting densities not yet supported for meta-analysis models.")
+      )
+    }
+
+    if (show_interval) {
+      # TODO: Handle NA boundaries
+      p <- p + geom_errorbar(aes(xmin = .data$CI_low, xmax = .data$CI_high),
+                             width = 0,
+                             size = size_point)
+    }
+
+    if (show_estimate) {
+      p <- p + geom_point(size = x$size_point * size_point, shape = x$shape)
+    }
+
+  } else if (isTRUE(multiple_ci)) {
 
     # plot setup for model parameters with multiple CIs
-    x <- datawizard::reshape_ci(x)
     x$CI <- as.character(x$CI)
-    p <- ggplot(x, aes(y = .data$Parameter, x = .data$Coefficient, color = .data$CI)) +
+
+    x$group <- factor(x$Coefficient < y_intercept, levels = c(FALSE, TRUE))
+    if (all(x$group == "TRUE")) {
+      color_scale <- scale_color_material(reverse = TRUE)
+    } else {
+      color_scale <- scale_color_material()
+    }
+
+    p <- ggplot(x, aes(y = .data$Parameter, x = .data$Coefficient,
+                       size = rev(.data$CI), color = .data$group)) +
       geom_vline(aes(xintercept = y_intercept), linetype = "dotted") +
-      geom_pointrange(
+      theme_modern(legend.position = "none") +
+      color_scale
+
+    if (show_density) {
+      p <- p + density_layer
+    }
+
+    if (show_interval) {
+      # TODO: Handle NA boundaries
+      p <- p + geom_errorbar(
         aes(xmin = .data$CI_low, xmax = .data$CI_high),
-        size = size_point,
-        position = position_dodge(1 / length(unique(x$CI)))
+        width = 0
       ) +
-      theme_modern() +
-      scale_color_material()
+        scale_size_ordinal(range = c(size_point, 3 * size_point))
+    }
+
+    if (show_estimate) {
+      p <- p + geom_point(
+        size = 4 * size_point
```

Comment:
Why do we have this strange internal conversion for point size? 

As a user, it would be difficult for me to figure out how to specify `size_point` argument to get the size I want.

Reply:
Previously, we used the `geom_pointrange()` function, which has this odd default that `size` refers to the linewidth and the point size is 4 times larger. I was mimicking that. We could change this to have size_line, size_point, and size_density arguments indpendently with the same defaults.

Reply:
I think I will adapt the `size` arguments here to match the ones used by ggdist.